src/main/java/com/microsoft/jenkins/containeragents/KubernetesCloud.java

```diff
@@ -401,6 +402,7 @@ public static String getServiceNameWithoutOrchestra(String serviceName) {
     }
 
     @Extension
+    @Symbol("azureKubernetes")
```

Comment:
Take care of the name here, there is a related one, [Azure Container Service (AKS) Plugin](https://github.com/jenkinsci/azure-acs-plugin).

Reply:
I can’t see a cloud in there, it just seemed to have a step called acsDeploy so should be fine unless I’m missing something

Reply:
I do cannot propose a better name which would deconflict plugins. Both focus the same feature, more or less. Suggestion below is a technical one which does not help too much

```suggestion
    @Symbol("azureKubernetesCloud")
```

Reply:
I would prefer not to include cloud in the same as it's part of the extension point.
i.e. for KubernetesCredentials the symbol would be kubernetes